### PR TITLE
hide minimum nodes alert (bsc#1046626)

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
I was not sure about the UI, maybe fadeOut/fadeIn would be better than hide/show here?

File: app/assets/javascripts/dashboard.js [hunk 1]
```diff
@@ -399,6 +399,17 @@ function toggleBootstrapButton() {
   $('#bootstrap').prop('disabled', !hasMinimumAmountToEnable);
 }
 
+// hide minimum nodes alert
+function hideMinimumNodesAlert() {
+  var hasMinimumNodesSelected = selectedNodesLength() > 2;
+
+  if (hasMinimumNodesSelected) {
+    $('.discovery-minimum-nodes-alert').hide();
```
Comment (vitoravelino) [inline app/assets/javascripts/dashboard.js]: I would use `fadeOut`.

File: app/assets/javascripts/dashboard.js [hunk 2]
```diff
@@ -399,6 +399,17 @@ function toggleBootstrapButton() {
...
+  } else {
+    $('.discovery-minimum-nodes-alert').show();
```
Comment (vitoravelino) [inline app/assets/javascripts/dashboard.js]: And here `fadeIn`.

File: app/assets/javascripts/dashboard.js [hunk 3]
```diff
@@ -399,6 +399,17 @@ function toggleBootstrapButton() {
   $('#bootstrap').prop('disabled', !hasMinimumAmountToEnable);
 }
 
+// hide minimum nodes alert
+function hideMinimumNodesAlert() {
```
Comment (vitoravelino) [inline app/assets/javascripts/dashboard.js]: Maybe renaming it to `toggleMinimumNodesAlert` would make more sense?

**Comment by vitoravelino** [comment]
> I was not sure about the UI, maybe fadeOut/fadeIn would be better than hide/show here?

KKK. I reviewed without reading this one. Yeah, i guess it's better.

**Comment by flavio** [comment]
ping @eotchi, to make sure she's fine with this change.

**Comment by eotchi** [comment]
Yep, fine by me.

**Pull request merged**